* fix default max. size for single log files to match the documentation.

Index: src/svlogd.c
===================================================================
RCS file: /var/lib/cvs/runit/src/svlogd.c,v
retrieving revision 1.5
diff -u -r1.5 svlogd.c
--- src/svlogd.c	3 May 2003 15:57:56 -0000	1.5
+++ src/svlogd.c	15 May 2003 09:21:00 -0000
@@ -305,7 +305,7 @@
   coe(ld->fdlock);
 
   ld->size =0;
-  ld->sizemax =9000;
+  ld->sizemax =1000000;
   ld->nmax =10;
   ld->name =(char*)fn;
   ld->match =0;
